parallel_algorithms/P3179/P3179.md

```diff
@@ -3343,11 +3358,6 @@ template<@_execution-policy_@ Ep, @_sized-random-access-range_@ R, class Proj =
 *Remarks*: If `f` returns a result, the result is ignored. Implementations do not have the freedom granted under
 [algorithms.parallel.exec]{- .sref} to make arbitrary copies of elements from the input sequence.
 
-[*Note X*: The overloads in namespace `ranges` require `Fun` to model `copy_constructible`. — *end note*]
-[LWG previously requested to remove the note about `Fun` modelling `copy_constructible`. However, it is a copy
-of the existing note for `ranges::for_each` that can be seen above. For consistency, should we keep or remove
-all these notes (also for `ranges::for_each_n`)?]{.draftnote audience="LWG"}
```

Comment:
So, what is the resolution of the consistency question? Seems only one note is removed but all others are kept - including a new such note in `for_each_n`?

Reply:
I don't remember that we discussed it for `for_each_n` specifically but the reasons mentioned during the discussion imply that we need to remove it here as well. Good catch!